worldedit-bukkit/adapters/adapter-1_17_1/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_17_R1_2/PaperweightGetBlocks.java

```diff
@@ -413,6 +413,7 @@ public LevelChunk ensureLoaded(ServerLevel nmsWorld, int chunkX, int chunkZ) {
     }
 
     @Override
+    @SuppressWarnings("rawtypes")
```

Comment:
Do we want to bother to extend this (etc.) to legacy adapters?

Reply:
I don't look into the adapters repo as often as I do in this one, I didn't bother with it much there.